image-loader/app/lib/ImageLoaderConfig.scala

```diff
@@ -26,4 +30,25 @@ class ImageLoaderConfig(override val configuration: Configuration) extends Commo
   val transcodedMimeTypes: List[MimeType] = getStringSetFromProperties("transcoded.mime.types").toList.map(MimeType(_))
   val supportedMimeTypes: List[MimeType] = List(Jpeg, Png) ::: transcodedMimeTypes
 
+  val imageProcessors: List[ImageProcessor] = {
+    val ImageProcessorClass: Regex = "class:(.*)".r
```

Comment:
Could we use the fully qualified class name in all cases and avoid the regex?

Reply:
A few reasons I did this:
 - I wanted to differentiate between built in logic and customer supplied logic. 
 - Where it's possible to do a compile time binding we should, even if the config is still used. 
 - This will also make it easier to figure out where the classes here are used.
 - Finally referencing a singleton object is non-obvious, but I guess we can sort that by converting the SupplierProcessors to a class.

Reply:
Makes sense. Maybe a `builtin:` prefix for the ones we have, just to make it clear? Not a big deal though

Reply:
I changed my mind on this.